jupyterlab_latex/build.py

```diff
@@ -11,7 +11,7 @@
 from .util import run_command
 
 @contextmanager
-def latex_cleanup(workdir='.', whitelist=None, greylist=None):
+def latex_cleanup(self, workdir='.', whitelist=None, greylist=None):
```

Comment:
Rather than passing `self` in here (which brings a lot of unnecessary API surface area to this function, can we pass in `cleanup`? That keeps the function signature a bit cleaner.`

Reply:
ok, since `self` is already in `get` it could get the setting there and pass it on.